server/controllers/auth.js

```diff
@@ -42,7 +42,8 @@ exports.registerUser = asyncHandler(async (req, res, next) => {
         user: {
           id: user._id,
           username: user.username,
-          email: user.email
+          email: user.email,
+          token: token
```

Comment:
Could you elaborate on the need for `token` here? I don't see this being used anywhere else in the PR.

Reply:
I just test and forgot to remove